src/calculator/stats.ts

```diff
@@ -1,64 +1,88 @@
 import { Stats } from '../entities/pokedex/types'
 
 export const getCpm = ( level: number ) => {
-  const memoized = levelToCpmMap.get( level )
+  const memoized = levelToCpmMap.get( level );
 
-  if( memoized ) return memoized
+  if( memoized ) return memoized;
 
   const cpm = (
-    level % 1 === 0 ?
+    level % 1.0 === 0 ?
```

Comment:
What is the purpose of the trailing zero? I'm not a fan but could be convinced if there's a good reason